presentation/views.py

```diff
@@ -34,3 +15,36 @@ class PresentationList(ListView):
 class PresentationDetail(DetailView):
     model = Presentation
     context_object_name = 'presentation'
+    template_name = 'presentation/presentation_list.html'
+
+
+def presentation_create(request):
+    form = PresentationCreateForm(request.POST)
+
+    if request.method == 'POST':
+        if form.is_valid():
+            try:
+                presentation = Presentation.objects.create(
+                    subject=form.cleaned_data.get('subject'),
+                    author=request.user,
+                    is_public=form.cleaned_data.get('is_public')
+                )
+
+                slide_list = request.POST.getlist('slide_list[]', [])
+
+                for slide in slide_list:
+                    Slide.objects.create(
+                        presentation=presentation,
+                        slide_order=slide['slide_order'],
+                        markdown=slide['markdown'],
+                        html=slide['html'],
+                    )
+
+                return redirect('list')
+            except Exception as e:
+                print(e)
```

Comment:
Do not use catch-print-ignore exception pattern. 
If you catch all of exceptions, print and do nothing, then we don't know there is some bugs. 
Instead of catch-print, just don't catch up, or catch-up and wrap that exception with our custom exception class. 
Then, django will catch the exception and display 500 error page to user and notify exception trace to us(e.g. Email or Slack).
The try-catch was made for recovering error and continuing business process. In the Web world, there is no way to do that.